.travis.yml

```diff
@@ -1,14 +1,17 @@
 language: android
+jdk: oraclejdk8
 android:
   components:
-    - build-tools-19.1.0
-    - android-19
+    - platform-tools
+    - tools
+    - build-tools-25.0.1
+    - android-25
+node_js:
```

Comment:
I was auditing other projects as well, and found your change 
https://github.com/appium/node-simctl/blob/master/.travis.yml
This might be more appropriate style for using multiple combinations. Do you have any recommendations for which versions of android do we intend to test against ?